apps/client/src/layout/components/breadcrumbs/hooks/use-crumbs.ts

```diff
@@ -0,0 +1,33 @@
+import invariant from 'tiny-invariant';
+import { useMatches } from 'react-router-dom';
+
+import {
+  isHandleableWithCrumbData,
+  isHandleableWithCrumbs,
+} from '../../../../helpers/predicates';
+import type {
+  MaybeCrumbly,
+  MaybeDataBound,
+  MaybeHandleable,
+} from '../../../../types';
```

Comment:
might be helpful to have the ability to use absolute paths.

Reply:
Yup! I missed some, but I did add "~/" as an alias to src/, matching the setup in IoT App Kit.